src/50_Visual_Effects/Snackbar_%2F_Toast.html

```diff
@@ -0,0 +1,107 @@
+<!---{
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  [Snackbars](https://material.io/guidelines/components/snackbars-toasts.html) provide brief feedback about an operation through a message at the bottom of the screen.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/../visual_effects/snackbar/toast/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <!-- ## Setup -->
+  <!-- We use amp-bind to trigger the Snackbar...  -->
```

Comment:
use code quotes for amp-bind and amp-animation

Reply:
done